src/Utility/DBConnection.java

```diff
@@ -0,0 +1,51 @@
+package Utility;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.apache.tomcat.jdbc.pool.DisposableConnectionFacade;
+
+public class DBConnection {
+	public static DBConnection dbConnection;
+	public static Connection connection;
+	public static PreparedStatement statement;
+	private DBConnection() {
+	
+	}
+	public static DBConnection getInstance() {
+		if(null==dbConnection) {
+			dbConnection=new DBConnection();
+			dbConnection.bdConnection();
+		}
+		return dbConnection;
+	}
+	public Connection bdConnection() {
+
+		try {
+			Class.forName("com.mysql.jdbc.Driver");
```

Comment:
Is it really necessary? If yes why?

Reply:
To check driver is register in jdbc sothat i can create database connection